README.md

```diff
@@ -140,6 +140,24 @@ make start-local-cluster
 make e2e-test
 ```
 
+### Release
+
+#### Configurator image
+The `newrelic-prometheus-configurator` images are pushed to [newrelic/newrelic-prometheus-configurator repo](https://hub.docker.com/r/newrelic/newrelic-prometheus-configurator). The following tags are pushed:
+- `main` each PR merged to the main branch will update this image.
+- `nightly` each night this image will be updated with the latest changes in the main branch. 
+- `latest` each Release will be update this image.
+- Release tag eg: `0.0.1`
+- Pre-Release tag eg: `0.0.1-pre`
+
+**Automated Release**:
```

Comment:
This is a release toolkit release, but it is not automated 😄

Reply:
is an automated release triggered manually. So the automation is:
- changelog
- release notes
- version number
and all of that is done by the RT